spec/models/account_spec.rb

```diff
@@ -14,47 +14,115 @@
   end
 
   describe "validations" do
-    it { is_expected.to validate_uniqueness_of(:synced_id) }
+    context "without multi applications support" do
+      it { is_expected.to validate_uniqueness_of(:synced_id) }
+    end
+
+    context "with multi applications support" do
+      before do
+        # TODO: Need to find a way to force this variable before the spec test run and the app code initialized
```

Comment:
how about just stubbing it?

Reply:
@Azdaroth Does not seem to work. Happy to be proven otherwise but could not make it work.

Reply:
Hmm, interesting. How did you stub it out? `allow(BookingSync::Engine).to receive(:multi_applications_support) { true }`?

Reply:
yes